tooling/sparta/README.md

```diff
@@ -270,3 +250,126 @@ module "sparta" {
 ## Support
 
 For support, please open an issue in the repository or contact the maintainers.
+
+## Monorepo Structure
+
+This project is now structured as a monorepo using Bun workspaces. Packages are located in the `packages/` directory. Dependencies are managed from the root `package.json`.
+
+To run commands within a specific package, use `bun run --filter <package-name> <script-name>`.
+
+## AWS DynamoDB Integration
+
+The application now uses AWS DynamoDB for session storage instead of in-memory storage. This provides several benefits:
+- Persistence across container restarts
+- Ability to scale horizontally with multiple instances
+- Improved reliability for session management
+
+### Local Development with DynamoDB
+
+For local development, you can use DynamoDB Local, which is provided by AWS as a downloadable version that simulates the DynamoDB service.
```

Comment:
Does this copy the database to the local machine?

Reply:
No, it just uses a local db. Copying would be a nice thing to do yeah. I'll look into it, I've done it ages ago so I def need a refresher